src/scripts_install/@DiceMetric

```diff
@@ -171,7 +171,7 @@ if ("$prefix" !~ "") then
    echo "#Col.0 = 'ROI label in $base ($lll)'" >> $prefix.1D
    set cnt = 1
    foreach dset ($dsets)
-      set bp = $base:h
+      set bp = `dirname $base`
```

Comment:
This seems fine, just surprised that this is done inside the loop over `dsets`, since `$base` is independent of that?